pyttb/sptensor.py

```diff
@@ -339,41 +422,52 @@ def from_aggregator(
         return cls(newsubs, newvals, shape, copy=False)
 
     def copy(self) -> sptensor:
-        """Make a deep copy of a :class:`pyttb.sptensor`.
+        """
+        Make a deep copy of an :class:`pyttb.sptensor`.
 
         Returns
         -------
         Copy of original sptensor.
 
         Examples
         --------
-        >>> first = ttb.sptensor(shape=(2,2))
-        >>> first[0,0] = 1
-        >>> second = first
-        >>> third = second.copy()
-        >>> first[0,0] = 3
-        >>> first[0,0] == second[0,0]
+        Create an :class:`pyttb.sptensor` (S1) and make a deep copy. Verify
+        the deep copy (S3) is not just a reference (like S2) to the original.
+
+        >>> S1 = ttb.sptensor(shape=(2,2))
+        >>> S1[0,0] = 1
+        >>> S2 = S1
+        >>> S3 = S1.copy()
+        >>> S1[0,0] = 3
+        >>> S1[0,0] == S2[0,0]
         True
-        >>> first[0,0] == third[0,0]
+        >>> S1[0,0] == S3[0,0]
         False
         """
         return ttb.sptensor(self.subs, self.vals, self.shape, copy=True)
 
     def __deepcopy__(self, memo):
         return self.copy()
 
-    # TODO decide if property
     def allsubs(self) -> np.ndarray:
         """
-        Generate all possible subscripts for sparse tensor
+        Generate all possible subscripts for sparse tensor.
```

Comment:
MATLAB is for math people (subs/subscripts) and Python is for CS people (indices). I got lost in trying to resolve these issues. The only place in `numpy` where subscripts are mentioned are in `einsum`. I started moving away from subscripts, but I think it would be better to align with MATLAB for now, then make a decision post-v2 if we want subscripts or indices.

Reply:
Sounds good. This is related to our previous discussion about making some note on unifying language/style. No change needed.

Reply:
Oops, I made all of these changes to subscripts in my local branch already before seeing this update. I think I'll make the change and we can standardize with a follow-up issue.